Makefile

```diff
@@ -40,9 +46,12 @@ clean:
 grind:
 	valgrind --leak-check=full node node_modules/.bin/_mocha
 
+# Note: this PATH setting is used to allow the localized tools to be used
+# but your locally installed osrm-backend tool, if on PATH should override
 shm: ./test/data/Makefile
-	$(MAKE) -C ./test/data
-	$(OSRM_DATASTORE) ./test/data/berlin-latest.osrm
+	@PATH="${PATH}:./lib/binding" && echo "*** Using osrm-datastore from `which osrm-datastore` ***"
+	@PATH="${PATH}:./lib/binding" && $(MAKE) -C ./test/data
+	@PATH="${PATH}:./lib/binding" && osrm-datastore ./test/data/berlin-latest.osrm
```

Comment:
If we add the local binaries to the front of `PATH` does bash prefer the local ones? Would this solve the issue?